### ISSUE TITLE
BUG: Admin can create lifecycle template with invalid data

### ISSUE THREAD

**Issue by Comoque1** [cherrytwist/Server#2062]
**Describe the bug**

**To Reproduce**
1. Go to http://localhost:3000/graphql
2. Use GA
3. Perform the following mutation
```
mutation createLifecycleTemplate($lifecycleTemplateInput: CreateLifecycleTemplateOnTemplatesSetInput!) {
  createLifecycleTemplate(lifecycleTemplateInput: $lifecycleTemplateInput) {
    info {
      title
      description
      tagset{tags}
    }
  }
}

{
  "lifecycleTemplateInput": {
      "templatesSetID": "05c1dae5-7da8-4847-97cd-e7be23ee47ab",
      "info": {
        "title": " ",
        "description": "",
        "tags": [""]
      },
      "type": "CHALLENGE",
      "definition": ""
  }
} 
```
4. See error - lifecycle template is created without type and empty definition and info
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/690a1c98-0083-4e3d-a254-0e98b92bf313)

**Expected behavior**
Validation for required fields is applied for lifecycle template.

**Comment by Comoque1** [comment]
There are disabled automated api tests.

**Comment by Comoque1** [comment]
There is disabled automated api test.

**Comment by techsmyth** [comment]
This is addressed with server #2046 i.e. when the LifecycleDefinition type is validated.
